src/System.Private.ServiceModel/src/Internals/WcfEventSource.cs

```diff
@@ -52,7 +52,7 @@ public bool ActionItemScheduledIsEnabled()
         }
 
         [Event(EventIds.ActionItemScheduled, Level = EventLevel.Verbose, Channel = EventChannel.Debug, Opcode = EventOpcode.Start, Task = Tasks.ThreadScheduling,
-            Keywords = Keywords.Threading,
+            Keywords = Keywords.Threading, ActivityOptions = EventActivityOptions.Disable,
```

Comment:
Nit: putting ActivityOptions on a new line would be more consistent with the formatting of the other parameters, but don't feel obligated to change unless you need to touch this file again for another reason.